### ISSUE TITLE
option to disable swiping up on spacebar

### ISSUE THREAD

**Issue by KraXen72** [dessalines/thumb-key#195]
- [x] Did you check to see if this issue already exists?
- [x] Is this only a single feature request? Do not put multiple feature requests in one issue.
- [x] Is this a question or discussion? Don't use this, use https://lemmy.ml/c/thumbkey

** Describe the feature request below **
most input boxes i type in are single line chat inputs. i often acidentally swipe on the spacebar up instead of tapping it, and it jumps to the start of the input where i continue typing without noticing. i could up the swipe threshold, but i have it set pretty low because of n and m, where i accidentally type n instead of m a lot.
please ade an option to disable vertical spacebar swiping, but keep horizontal bc it's useful. thanks

**Comment by dessalines** [comment]
Good call... I'll disable the up / down actions on the main spacebar. The programming layouts will use their own special keyboard.

**Comment by KraXen72** [comment]
i think moving that functionality to the enter key should be a pretty good idea. also full words movement left & right could be on the enter key. (i'm still hoping to see precise drag-based letter moving on spacebar: move by how much you drag)

**Comment by KraXen72** [comment]
im going to add some keymaps, i can add it to the env4 and if you like it you could just copy it to the other layouts? or do you want to handle this somewhere else than in the keymaps themselves?

**Comment by dessalines** [comment]
No, not the enter key, the spacebar moving comes from messageease, and it makes sense. 

The "back word" actions also don't make much sense to me. If you're going back many words, you should just press that in the text area, and if you want to delete a word, thumbkey already has that on the swipe left delete key.

> im going to add some keymaps, i can add it to the env4 and if you like it you could just copy it to the other layouts? or do you want to handle this somewhere else than in the keymaps themselves?

Yes, or just create a new file, or copy-paste it somewhere.
